MtfhReportingDataListener/serverless.yml

```diff
@@ -27,11 +27,12 @@ functions:
       DATAPLATFORM_KAFKA_HOSTNAME: ${ssm:/housing-tl/${self:provider.stage}/dataplatform-kafka-hostname}
       SCHEMA_ARN: ${ssm:/housing-tl/${self:provider.stage}/dataplatform-tenure-schema-arn}
       ROLE_ARN_TO_ACCESS_DATAPLATFORM_GLUE_REGISTRY: ${ssm:/housing-tl/${self:provider.stage}/dataplatform-role-arn-to-access-glue-registry}
+      TENURE_API_KAFKA_TOPIC: ${ssm:/housing-tl/${self:provider.stage}/tenure-kafka-topic}
```

Comment:
Has this been created in aws parameter store?

Reply:
I don't believe so. We don't have access to the housing AWS accounts, so it will need to be created if it doesn't already been exist

Reply:
I've created the value in aws parameter store - `/housing-tl/${self:provider.stage}/dataplatform-kafka-topic`